app/email_utils.py

```diff
@@ -19,7 +19,7 @@
 from typing import Tuple, List, Optional, Union
 
 import arrow
-import dkim
+import dkim  # pylint: disable=syntax-error
```

Comment:
why do we need to add `disable=syntax-error` here?

Reply:
Because `dkimpy` has a `# type:` comment that should be placed before the docstring. Otherwise pylint detects it as an error unless we add this annotation.

```
def verify(message, logger=None, dnsfunc=get_txt, minkey=1024,
        timeout=5, tlsrpt=False):
    """Verify the first (topmost) DKIM signature on an RFC822 formatted message.
    @param message: an RFC822 formatted message (with either \\n or \\r\\n line endings)
    @param logger: a logger to which debug info will be written (default None)
    @param timeout: number of seconds for DNS lookup timeout (default = 5)
    @param tlsrpt: message is an RFC 8460 TLS report (default False)
     False: Not a tlsrpt, True: Is a tlsrpt, 'strict': tlsrpt, invalid if
     service type is missing. For signing, if True, length is never used.
    @return: True if signature verifies or False otherwise
    """
    # type: (bytes, any, function, int) -> bool
```

Reply:
ah ok I see